Outdated documentation setting up a test app (#3511)
grafana/tempo: docs/sources/tempo/setup/set-up-test-app.md

1. Install `telemetrygen` using the [installation procedure](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen).
   **NOTE**: You don't need to configure an OpenTelemetry Collector as we are using the Grafana Agent.

> 2. Generate traces using `telemtrygen`:

Reviewer: ```suggestion
2. Generate traces using `telemetrygen`:
```

   ```bash
   telemetrygen traces --otlp-insecure --rate 20 --duration 5s --otlp-endpoint grafana-agent.grafana-agent.svc.cluster.local:4317
   ```
  This configuration sends traces to Grafana Agent for 5 seconds, at a rate of 20 traces per second.

  Optionally, you can also send the trace directly to the Tempo database without using Grafana Agent as a collector by using the following:
  ```bash
  telemetrygen traces --otlp-insecure --rate 20 --duration 5s --otlp-endpoint tempo-cluster-distributor.tempo.svc.cluster.local:4317
  ```

>   If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, eg:

Reviewer: ```suggestion
  If you're running `telemetrygen` on your local machine, ensure that you first port-forward to the relevant Agent or Tempo distributor service, for example:
```

  ```bash
  kubectl port-forward services/grafana-agent 4317:4317 --namespace grafana-agent
  ```

> 3. Alertnatively a cronjob can be created to send traces periodically based on this template

Reviewer 2: ```suggestion
3. Alternatively a cronjob can be created to send traces periodically based on this template
```

> 3. Alternatively a cronjob can be created to send traces periodically based on this template

Reviewer: ```suggestion
3. Alternatively, a cronjob can be created to send traces periodically based on this template:
```
